homeassistant/components/roon/media_player.py

```diff
@@ -124,6 +122,8 @@ def __init__(self, server, player_data):
         self._attr_shuffle = False
         self._attr_media_image_url = None
         self._attr_volume_level = 0
+        self._attr_volume_fixed = True
+        self._attr_volume_incremental = False
```

Comment:
Please give other names to these to not confuse with the `_attr-` used as shorthand for entity properties